rules/combo/backdoor/php.yara

```diff
@@ -116,8 +116,13 @@ rule php_eval_gzinflate_base64_backdoor : critical {
     $f_gzinflate = "gzinflate("
     $f_base64_decode = "base64_decode"
 
-    $not_php = "PHP_FLOAT_DIG" fullword
+    $not_js = " ?? "
```

Comment:
We also do this for the `base64_python_functions` rule where JS code was being evaluated in the context of Python code.